reference/config_files/global_conf.rst

```diff
@@ -0,0 +1,290 @@
+.. _reference_config_files_global_conf:
+
+global.conf
+===========
+
+The **global.conf** file is located in the Conan user home directory, e.g., *[CONAN_HOME]/global.conf*.
+
+Introduction to configuration
+-----------------------------
+
+*global.conf* is aimed to save some core/tools configuration variables that will be used by Conan. For instance:
```

Comment:
I think it's worth explaining what's the difference about the core config and the tools config

Reply:
Yes! Agree 👍